variables.tf

```diff
@@ -0,0 +1,90 @@
+variable "resource_group_name" {
+  type    = string
+  default = "aks-rg"
+}
+
+variable "resource_group_location" {
+  type    = string
+  default = "East US"
+}
+
+variable "cluster_name" {
+  type    = string
+  default = "akscluster"
+}
+
+variable "dns_prefix" {
+  description = "(Required) DNS prefix specified when creating the managed cluster. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "service_principal_id" {
+  type = string
+}
+
+variable "service_principal_secret" {
+  type = string
+}
+
+variable "node_count" {
+  type    = string
+  default = "1"
+}
+
+variable "vm_size" {
+  type    = string
+  default = "Standard_D2s_v3"
+}
+
+variable "kubernetes_version" {
+  default = "1.15.7"
+  type    = string
+}
+
+variable "admin_user" {
+  type    = string
+  default = "aksadmin"
+}
+
+variable "ssh_public_key" {
+  type        = string
+  default     = "~/.ssh/id_rsa.pub"
```

Comment:
Does this mean it is using your public ssh key and that it is a requirement to have one in the default location?